locale/tr/LC_MESSAGES/More/Repository/README.po

```diff
@@ -50,7 +50,7 @@ msgstr ""
 
 #: ../../More/Repository/README.rst:20
 msgid "Laravel Framework"
-msgstr ""
+msgstr "Aptal bir framework."
```

Comment:
why translate Laravel Framework? It's a name.

Reply:
What! Translation says, 'It is a stupid framework.' ??